examples/api/version.feature

```diff
@@ -10,7 +10,7 @@ Feature: get version
     And the response should match json:
       """
       {
-        "error": "Method not allowed"
+        "error": "ethod not allowed"
```

Comment:
fix typo `method`

Reply:
yes thanks, but most likely this pull request won't be merged, as all hooks will be refactored at some point to return an error, to follow the idiomatic go way, see #88 that way it will be possible to simple extend error message in a hook.

Reply:
Glad to hear! As a STE automating acceptance criteria getting meaningful message is important for debugging. currently it's hard with current dodog version.